spring-cloud-contract-stub-runner/src/test/groovy/org/springframework/cloud/contract/stubrunner/AetherStubDownloaderSpec.groovy

```diff
@@ -29,9 +28,6 @@ import org.springframework.util.ResourceUtils
 
 class AetherStubDownloaderSpec extends Specification {
 
-	@Rule
```

Comment:
I don't really know why this rule was here in the first place, we were not using Hoverfly in any of the tests from what I can tell